### PR TITLE
Bug bash: fix transform issues

### PR DESCRIPTION
**Pull request by chenriksson**
Fixes for the following:
- #5311: Transform should fail if admin is same as transformed account
- #5309: Transform should fail if admin is an organization
- #5317: Transform error not formatted correctly for unconfirmed admin

**Update 1/23**: Also incorporating feedback from @anangaur, including:
- #5308 and other transform error messages
- UI: Change admin info message to tooltip
- UI: Add transform confirmation
- UI: Formatting of sign-in returnUrl message

### CODE REVIEW

File: src/NuGetGallery/Services/UserService.cs [hunk 1]
```diff
@@ -214,7 +214,33 @@ public bool CanTransformUserToOrganization(User accountToTransform, out string e
...
+            else if (adminUser is Organization)
+            {
+                errorReason = String.Format(CultureInfo.CurrentCulture,
+                    Strings.TransformAccount_AdminAccountIsOrganization, adminUser.Username);
+            }
+
+            return errorReason == null;
+        }
+
+            public async Task<bool> TransformUserToOrganization(User accountToTransform, User adminUser, string token)
```
Comment (loic-sharma) [inline src/NuGetGallery/Services/UserService.cs]: Spacing seems off here

Comment (chenriksson) [reply src/NuGetGallery/Services/UserService.cs]: Fixed

File: src/NuGetGallery/Views/Shared/Gallery/Header.cshtml [hunk 2]
```diff
@@ -151,7 +151,7 @@
     <div class="alert-transient @(ViewBag.HasJumbotron == true ? "alert-transient-jumbotron" : string.Empty)">
         <div class="container">
             <div class="row">
-                <div class="@ViewHelpers.GetColumnClasses(ViewBag)" role="alert" aria-live="assertive">
+                <div class="col-sm-12" role="alert" aria-live="assertive">
```
Comment (joelverhagen) [inline src/NuGetGallery/Views/Shared/Gallery/Header.cshtml]: Why this change?

Comment (joelverhagen) [reply src/NuGetGallery/Views/Shared/Gallery/Header.cshtml]: If I recall, not all pages take up all columns (i.e. some page content uses less than the 12 columns at which point we wanted the temp message to be that smaller width).

File: src/NuGetGallery/Views/Users/Transform.cshtml [hunk 3]
```diff
@@ -60,4 +61,19 @@
...
     </div>
-</section>
\ No newline at end of file
+</section>
+
+@section bottomScripts {
+    <script type="text/javascript">
+        $(function ()
+        {
+           var confirmation = "@Html.Raw(transformConfirmation)";
```
Comment (joelverhagen) [inline src/NuGetGallery/Views/Users/Transform.cshtml]: You can JSON encode the string instead of doing raw.

File: src/NuGetGallery/Views/Shared/Gallery/Header.cshtml (hunk 2 above)
Comment (chenriksson) [reply src/NuGetGallery/Views/Shared/Gallery/Header.cshtml]: Per @anangaur 's feedback that the temp message should stretch the width of the header. We were noticing the temp message width vary by page, which seems consistent with what you're saying.

I can circle back w/ him to verify.

Comment (joelverhagen) [reply src/NuGetGallery/Views/Shared/Gallery/Header.cshtml]: Okay, if this is the case please verify we don't have dead code. I don't feel strongly about it but when we worked with @jonwchu we decided to make the transient message width match the page content width. I don't feel strongly.

File: src/NuGetGallery/Views/Users/Transform.cshtml (hunk 3 above)
Comment (chenriksson) [reply src/NuGetGallery/Views/Users/Transform.cshtml]: I did `@Html.Raw` to prevent encoding.

File: src/NuGetGallery/Views/Shared/Gallery/Header.cshtml (hunk 2 above)
Comment (chenriksson) [reply src/NuGetGallery/Views/Shared/Gallery/Header.cshtml]: Reverting, will open issue and can sort out later at lower pri.
